src/components/Picker/Picker.js

```diff
@@ -169,6 +166,34 @@ class Picker extends PureComponent {
         });
     }
 
+    /**
+     * This method is used by Form
+     */
+    // eslint-disable-next-line rulesdir/prefer-early-return
+    blur() {
+        if (Platform.OS === 'web') {
+            this.picker.blur();
+        }
+    }
+
+    /**
+     * This method is used by Form
+     */
+    // eslint-disable-next-line rulesdir/prefer-early-return
+    focus() {
+        // On the Web, calling `focus` improves accessibility. It doesn't open the picker.
+        if (Platform.OS === 'web') {
+            this.picker.focus();
+        }
+    }
+
+    /**
+     * This method is used by Form when scrolling to the input
+     */
+    measureLayout(...args) {
+        return this.textInput.measureLayout(...args);
```

Comment:
You're right, it just doesn't. I haven't tested this newest commit on Web. I screwed up.

> Remove all focus, blur and measureLayout functions.

Please review [this thread](https://github.com/Expensify/App/pull/16433#discussion_r1147843792) on the conceptual level! 😇 Please.

I can remove and add any methods you want, but this code was like that for a reason (and I'm _not_ talking about platform switching vs platform files. I'm sorry I missed that).

Reply:
What would be the contract of that `getPickerRef` method? The best one I could figure out is:

```
/**
 * Returns a platform-dependent reference, which on Web exposes `focus`, but doesn't expose `measureLayout`,
 * while on Native is exposes `measureLayout`, but a non-working `focus`.
*/
```

I want to use `measureLayout` cross-platform, because it gives [**consistent results**](https://github.com/Expensify/App/pull/16433#issuecomment-1481024942). It's a user-perceived thing, not a technicality.

Reply:
> and to pickerProps on Web then it will have measureLayout.
Have you checked that?

Reply:
```js
                        pickerProps={{
                            ref: (el) => {
                                console.log({
                                    measureLayout: el.measureLayout,
                                });
                            },
```

![image](https://user-images.githubusercontent.com/2590174/227735901-0f8a56f1-0fe4-434f-beb7-51a0a0178ec2.png)

Reply:
I don't argue with the `Platform.OS`. It's a different topic.

In the newest push I:

- Fixed the screw-up with using `textInput` on Web (in the simplest way I could think of, i.e. by not using that reference...)
- Switched to platform files to follow our conventions

I did that by adding a boolean flag. This approach could be re-considered, i.e. maybe there could be some other way than a boolean flag. But let's leave that for later, if you'd agree.

At this point, I believe that:
* I can switch (again) to using `TextInput`, but **this will add user-perceived regression**. Scrolling will behave inconsistently for pickers and other inputs. I'm not saying **not to do that**, but I'm insisting that we talk about user-perceived effects of that **first** and then about code organization.

Reply:
I tried to cover this whole problem [here](https://github.com/Expensify/App/pull/16433#issuecomment-1481024942), with videos of incosistent behavior resulting from the lack of `measureLayout` on the `react-native-picker/picker`.

Reply:
@parasharrajat But if you don't think that that incosistency is a big deal (that we don't need/want do layout measuring on Web) I can do that picker/TextInput dispatch, I guess. I don't like the fact that it affects the user, but I also wouldn't like to miss the bonus 🙁